src/modules/ItineraryMakerModule/index.tsx

```
@@ -16,6 +16,10 @@ import {
   Loader2,
   Lightbulb,
   Wand2,
+  ListChecks,
+  Map,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)